_editorial_policies/02_editor_instructions.md

```diff
@@ -11,28 +11,32 @@ This provides instructions to editors handling manuscripts for LiveCoMS.
 
 ## Workflow overview
 
-Authors contact LiveCoMS (specifically, the Lead Editor in a particular subject area) with a presubmission letter, proposing an article in a particular area.
-Once this is approved, they proceed with article preparation and ultimately submit it to that section of LiveCoMS.
-The Lead Editor then checks that the article is consistent with what was proposed in the presubmission letter, and then assigns it to an appropriate Associate Editor who manages the review process and makes a final decision on the manuscript, reporting the final decision to the authors.
+Authors contact LiveCoMS (specifically, the Section Lead Editor reponsible for the corresponding article type) with a presubmission letter, proposing an article in a particular topic area. The Section Lead Editor assigns this presubmission letter to an Associate Editor, who works with the authors to revise the article plans to be suitable for LivCoMS, and the Section Lead Editor signs off on the final plans.
+Once the presubmission letter is approved, the author proceeds with article preparation and ultimately submits the article to LiveCoMS.
+The Lead Editor then checks that the article is consistent with what was proposed in the presubmission letter, and assigns it to an appropriate Associate Editor who manages the review process and makes a final decision on the manuscript, reporting the final decision to the authors. Usually this Associate Editor will be the same one that handled the original presubmission letters.
+
+## Pre-submission letters
+
+Assuming the subject matter proposed in the presubmission letter is within the scope of LiveCoMS and is otherwise acceptable, the Section Lead Editor assigns this presubmission letter to an Associate Editor. Reasons for the letter not being acceptable include, for example, ignoring key required elements of the presubmission letter or being too similar to existing LiveCoMS articles. The Section Lead Editor signs off on the final presubmission letter, and the Associate Editor notifies the author that the letter is approved.  Instructions for the presubmission letter are [found here](https://github.com/livecomsjournal/livecomsjournal.github.io).
```

Comment:
see my prior comment

Reply:
I'm not sure which one you'd like me to see.  Could you be more specific?

Reply:
I can't remember now, but I think I was worried about inconsistency between this spot and another (role of assoc editors?) on the workflow for approving presubmission letter - whether section lead approval was needed in every case or only special cases.

Reply:
Maybe if it was changed earlier from "work with authors to develop presubmission letters" rather than "work with authors to finalize presubmission letters" it would be more consistent?  I think that actually the guidance of section lead editors is more important at the presubmission phase to make sure the scope is good.